### PR TITLE
Refactor projections

### LINKED ISSUE #505: Verify image projections
https://neubias.github.io/training-resources/projections/index.html

* tiff reader

### PR DESCRIPTION
**Pull request by tibuch**
Closes #505.

* Use OpenIJTIFF
* Use multiactivities

### CODE REVIEW

File: _includes/projections/projections_skimage_napari.py [hunk 1]
```diff
@@ -1,43 +1,60 @@
...
 
+#%%
 # Maximum projection along x-axis and y-axis
 max_x_image = np.max(image, axis=2)
 max_y_image = np.max(image, axis=1)
 viewer.add_image(max_x_image)
 viewer.add_image(max_y_image)
 
+#%%
 # Due to the aniotropic pixel size the x and y projections appear squashed
```
Comment (grinic) [inline _includes/projections/projections_skimage_napari.py:48]:
```suggestion
# Due to the anisotropic pixel size, the x and y projections appear squashed
```

**Comment by manerotoni** [comment]
Hello @tibuch, hello @grinic, hello @tischi,
I am going through the module. In the sum projection exercise how do the sum image look like on Napari?

In my hands Napari has a problem properly rendering uint32 properly.  It works fine if I convert to float first. The left image is the conversion, the right is the uint32. They are shown in the same range. 

image

File: _includes/projections/projections_skimage_napari.py (hunk 1 above)
Comment (manerotoni) [reply _includes/projections/projections_skimage_napari.py:48]: I will include your changes in the next commits

**Comment by manerotoni** [comment]
Hi @tibuch,
I did some refactoring of the activity and renaming of the files so that they are consistent with the rest of the material. 
I checked if it all worked. 
You can merge.

**Comment by tibuch** [comment]
Hi @manerotoni,

Thanks for all the fixes! Should we now link to the volume slicing module in the prerequisites?

**Comment by manerotoni** [comment]
> Thanks for all the fixes! Should we now link to the volume slicing module in the prerequisites?

May be the [N-dimensional images](https://neubias.github.io/training-resources/multidimensional_image_basics/index.html) module ?
Volume slicing is special and one can do projections without slicing? https://neubias.github.io/training-resources/volume_slicing/index.html

**Comment by tibuch** [comment]
True, that makes more sense.

**Pull request merged**